metadata/sources.go

```diff
@@ -0,0 +1,39 @@
+package metadata
+
+type (
```

Comment:
We could create a package called `sources` for this file

Reply:
👍